Application/mobile/app_code/test/theme_test.dart

```diff
@@ -19,7 +21,7 @@ void main() {
     testWidgets('Toggle theme and save to SharedPreferences', (tester) async {
       SharedPreferences.setMockInitialValues({'appTheme': appTheme['clair']});
 
-      await tester.pumpWidget(
+      await tester.pumpWidget(initPage(
```

Comment:
That's not how we use initPage, read the function or ask question if you don't understand